### ISSUE TITLE
Separate notification channel for VIP expiry dates

### ISSUE THREAD

**Issue by RobinDadswell** [Prowlarr/Prowlarr#103]
Ability to send notifications specifically for VIP expiry dates, this is currently included within HealthChecks but should be able to be sent as it's own type of notification as well.

**Comment by Qstick** [comment]
Good call, we should create indexer notifications such as VIP expiration, grab limits, api limits reached, etc…

**Comment by RobinDadswell** [comment]
Yeah, probably a good shout for all of those; I feel like they should be health checks as well as they are obvious but have the ability to send certain ones without all the health checks (like the ones you listed)
